### PR TITLE
Video update

### PR DESCRIPTION
**Pull request by ace-n**
**Do not merge** until the TODOs are resolved.

### CODE REVIEW

File: video/analyze.js [hunk 1]
```diff
@@ -235,9 +259,9 @@ function analyzeSafeSearch (gcsUri) {
...
     .then((results) => {
       // Gets unsafe content
-      const safeSearchResults = results[0].annotationResults[0].safeSearchAnnotations;
-      console.log('Safe search results:');
-      safeSearchResults.forEach((result) => {
+      const explicitContentResults = results[0].annotationResults[0].safeSearchAnnotations;
+      console.log('Explicit content results:');
+      explicitContentResults.forEach((result) => {
         console.log(`Time: ${result.timeOffset / 1e6}s`);
         console.log(`\tAdult: ${likelihoods[result.adult]}`);
```
Comment (dizcology) [inline video/analyze.js]: Spoof, medical etc are removed (the API has not been returning non-empty values for those), and only adult (renamed pornography_likelihood) remains.

File: video/system-test/analyze.test.js [hunk 2]
```diff
@@ -24,50 +24,58 @@ const tools = require(`@google-cloud/nodejs-repo-tools`);
...
 });
 
 // analyze_labels_gcs (one scene)
-test(`should analyze labels in a GCS file with one scene`, async (t) => {
-  const output = await tools.runAsync(`${cmd} labels-gcs gs://demomaker/tomatoes.mp4`, cwd);
-  t.regex(output, /Label Tomato occurs at:/);
+test.serial(`should analyze labels in a GCS file with one scene`, async (t) => {
+  const output = await tools.runAsync(`${cmd} labels-gcs ${shortUrl}`, cwd);
+  t.regex(output, /Label shirt occurs at:/);
   t.regex(output, /Entire video/);
```
Comment (dizcology) [inline video/system-test/analyze.test.js]: We might no longer show "Entire video" in the output.  (see comments above about time offsets)

**Comment by gguuss** [comment]
@jabubake I think we're good to merge when the tests come back green. They're working locally for me with your fixes.

**Comment by gguuss** [comment]
@jabubake Wrong PR, [this one](https://github.com/GoogleCloudPlatform/java-docs-samples/pull/849) is the one that's ready to merge.

**Comment by gguuss** [comment]
@jmdobry I think we're ready to merge when the tests are green.

**Comment by jmdobry** [comment]
@dizcology has a bunch of outstanding comments that need to be addressed.

**Comment by gguuss** [comment]
Addressing the TODOs now, missed that.

**Comment by gguuss** [comment]
I should have addressed all review feedback at this point.

**Pull request merged**